### ISSUE TITLE
missing riskScore condition in okta_app_signon_policy_rule 

### ISSUE THREAD

**Issue by eduhamel** [okta/terraform-provider-okta#1284]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The riskScore condition is missing from the okta_app_signon_policy_rule
It is not retrieved when importing a rule from an authentication policy, and it doesn't seem to be possible to configure it either when pushing a rule on Okta.

### New or Affected Resource(s)

- okta_app_signon_policy_rule 

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

- #0000

**Comment by monde** [comment]
Okta internal reference https://oktainc.atlassian.net/browse/OKTA-531875

**Comment by monde** [comment]
@eduhamel I'll also call this a bug, it needs to be corrected.

**Comment by BartBartBartBartBart** [comment]
Not stale

**Comment by BartBartBartBartBart** [comment]
Can this issue be reopened? 
Risk score is a very important feature in Okta Application policies and it is not available in the provider.

**Comment by duytiennguyen-okta** [comment]
Closed with #1780
